src/main/resources/cucumber-junit-runner.vm

```diff
@@ -2,13 +2,17 @@ import org.junit.runner.RunWith;
 
 import cucumber.api.CucumberOptions;
 import cucumber.api.junit.Cucumber;
-
+#set( $condition = $generateByLineNumber)
 @RunWith(Cucumber.class)
 @CucumberOptions(strict = $strict,
-features = {"$featureFile"},
+#if ($condition)
```

Comment:
I don't think this should be put in the template.

It is overly complicating it and means it would have to be applied in multiple places for each template.